### PR TITLE
test(protocol-designer): turn on cypress e2e migration tests

### LINKED ISSUE #9852: PD: remove skipped migration e2e tests
# Overview

Currently PD migration e2e tests (`protocol-designer/cypress/integration/migrations.spec.js`) are being skipped while JSON schema v6 support gets fully implemented. Once the [liquids key](https://github.com/Opentrons/opentrons/issues/9702) is added to PD, we should be able to generate new v6 fixtures and un skip the migration tests. 

Note: previously migrations would conditionally bump schema versions, now we ALWAYS bump to schema v6, so our new assertion fixtures must reflect this. 

# Acceptance criteria

PD migration tests are running and passing

### PR DESCRIPTION
**Pull request by shlokamin**
# Overview
This PR fixes and turns back on protocol designer's cypress e2e migration tests.

closes https://github.com/Opentrons/opentrons/issues/9852

# Changelog

- Turn on cypress e2e migration tests

# Review requests

Migration tests should be on and passing! You can check CI to verify

# Risk assessment

Low

### CODE REVIEW

File: protocol-designer/src/components/FileSidebar/FileSidebar.tsx [hunk 1]
```diff
@@ -16,7 +16,7 @@ import modalStyles from '../modals/modal.css'
 import styles from './FileSidebar.css'
 
 import { HintKey } from '../../tutorial'
-import {
+import type {
```
Comment (shlokamin) [inline protocol-designer/src/components/FileSidebar/FileSidebar.tsx:19]: unrelated to e2e tests, i just saw this and i wanted to fix it lol

Comment (reachojhaneha) [reply protocol-designer/src/components/FileSidebar/FileSidebar.tsx:19]: why is this added?

**Review by reachojhaneha — APPROVED** [review]
left a comment about importing Type. other than that LGTM

File: protocol-designer/src/components/FileSidebar/FileSidebar.tsx (hunk 1 above)
Comment (shlokamin) [reply protocol-designer/src/components/FileSidebar/FileSidebar.tsx:19]: can u see the comment above? https://github.com/Opentrons/opentrons/pull/10849/files#r902940733

**Pull request merged**